README.md

```diff
@@ -233,6 +233,15 @@ If you used Windows and decided to switch to Linux, this course should help you
 * [Bash Guide for Beginners](https://linux.die.net/Bash-Beginners-Guide/)
 * [Advanced Bash-Scripting Guide](https://linux.die.net/abs-guide/)
 
+### ZSH Useful Resources
+* [antigen](https://github.com/zsh-users/antigen)
+* [oh-my-zsh](http://ohmyz.sh/)
```

Comment:
I wouldn't really call `oh-my-zsh` and `antigen` learning resources. Awesome? Definitely. But not learning resources.

Reply:
Ok, I'll remove these two